x-pack/legacy/plugins/apm/public/components/app/ServiceMap/cytoscapeOptions.ts

```diff
@@ -15,22 +15,18 @@ export const animationOptions: cytoscape.AnimationOptions = {
 const lineColor = '#C5CCD7';
 export const nodeHeight = parseInt(theme.avatarSizing.l.size, 10);
 
-const layout = {
-  name: 'dagre',
-  nodeDimensionsIncludeLabels: true,
-  rankDir: 'LR',
-  animate: true,
-  animationEasing: animationOptions.easing,
-  animationDuration: animationOptions.duration,
-  fit: true,
-  padding: nodeHeight
-};
-
 function isService(el: cytoscape.NodeSingular) {
   return el.data('type') === 'service';
 }
 
 const style: cytoscape.Stylesheet[] = [
+  // @ts-ignore
```

Comment:
If we update to the latest `@types/cytoscape` some of these will go away. We should. I know `visibility` is in there as of the latest.